wagtail/admin/templates/wagtailadmin/shared/headers/slim_header.html

```diff
@@ -58,21 +58,37 @@ <h1 class="w-sr-only">
             </div>
 
             {% block search_form %}
-                {% if search_url and search_form %}
+                {% if not search_url %}
+                {% elif search_form or filters %}
```

Comment:
Empty if, to avoid extra indent

The `<form>` now works with `search_form` or `filters` (or both). You can have filters without search and vice versa, but use the same URL in general (the index results URL)